src/prefect/tasks/snowflake/snowflake.py

```diff
@@ -129,11 +131,11 @@ def run(
                 takes snowflake AUTOCOMMIT parameter
             - cursor_type (SnowflakeCursor, optional): specify the type of database
                 cursor to use for the query, defaults to SnowflakeCursor
-        - authenticator (str, optional): type of authenticator to use for initiating
-            connection (oauth, externalbrowser...), refer to snowflake documentation
-            https://docs.snowflake.com/en/user-guide/python-connector-api.html#connect
-            for details, note that `externalbrowser` will only work in an environment
-            where a browser is available, default to None
+            - authenticator (str, optional): type of authenticator to use for initiating
+                connection (oauth, externalbrowser...), refer to snowflake documentation
+                https://docs.snowflake.com/en/user-guide/python-connector-api.html#connect
```

Comment:
You can format the link as a markdown link. I would use something like this:

"refer to Snowflake [Python Connector API](https://docs.snowflake.com/en/user-guide/python-connector-api.html#connect) documentation for details. Note that..."